src/containerbuddy/config.go

```diff
@@ -65,6 +65,7 @@ type ServiceConfig struct {
 	Port             int             `json:"port"`
 	TTL              int             `json:"ttl"`
 	Interfaces       json.RawMessage `json:"interfaces"`
+	Tags             []string        `json:"tags,omitempty"`
```

Comment:
I think we're going to need to include tags for the `BackendConfig` as well. It's my understanding that the primary use case for this that @misterbisson was talking about in #10 was being able to split out services for e.g. dev vs prod. So when we query Consul we're probably going to want to have a list of tags to send to [`checkHealth`](https://github.com/joyent/containerbuddy/blob/7249deb6e69fd66469a3f4ed366186d28937769b/src/containerbuddy/consul.go#L84) and then on to [`Health.Service`](https://godoc.org/github.com/hashicorp/consul/api#Health.Service)